cmd/clusterctl/cmd/config_repositories.go

```diff
@@ -40,15 +43,18 @@ var configRepositoryCmd = &cobra.Command{
 		clusterctl config repositories`),
 
 	RunE: func(cmd *cobra.Command, args []string) error {
-		return runGetRepositories()
+		return runGetRepositories(cfgFile, os.Stdout)
```

Comment:
Was this done in order to test this function?

Reply:
Yeah, pretty much.